### PR TITLE
Manual updates

### PR DESCRIPTION
**Pull request by raoanirudh**
Addresses more items from #3460. Build: https://ci.openquake.org/job/builders/job/pdf-builder/187

- Update intro page
- Remove unused appendix dir+file
- Mention that exposure can be provided as a zip file
- Document additional params for scenario_risk
- Document additional params for event_based_risk
- Update scenario_damage outputs
- Describe newly added dmg_by_event scenario output
- Remove asset_loss_table parameter

### CODE REVIEW

**Comment by micheles** [comment]
Build red, you should bug @vot4anto

**Comment by raoanirudh** [comment]
> Build red, you should bug @vot4anto

Passing now

**Review by micheles — APPROVED** [review]

File: doc/manual/oqum/risk/02_config_event_based_risk.tex [hunk 1]
```diff
@@ -124,3 +124,69 @@
...
+    zero and one are no longer supported due to performance considerations. The
+    only two values permitted are \Verb+asset_correlation = 0+ and 
+    \Verb+asset_correlation = 1+.
+
+  \item \Verb+asset_loss_table+: this parameter
+    specifies whether the individual asset losses should be stored for every 
+    event for each realization; the default value of this parameter is
+    \Verb+false+. If this flag is set to \Verb+true+, a
+    matrix containing all of the losses is saved in the datastore.
+
```
Comment (micheles) [inline doc/manual/oqum/risk/02_config_event_based_risk.tex:172]: I am not sure if I want to restore this parameter. For sure, I do not want it documented.

Comment (raoanirudh) [reply doc/manual/oqum/risk/02_config_event_based_risk.tex]: Removed asset_loss_table from both scenario_risk and event_based_risk sections

**Pull request merged**